prisma/migrations/20230712130728_add_new_fields_to_some_tables/migration.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
+
+  - Added the required column `phoneNumber` to the `Info` table without a default value. This is not possible if the table is not empty.
+
+*/
```

Comment:
Deu uma avaliada nesse warning já? Acha que seria uma boa deixar algum valor default?

Reply:
I add a varChar limit, no make sense put a default value because this value is required